include/deal.II/numerics/vector_tools_interpolate.h

```diff
@@ -284,6 +284,16 @@ namespace VectorTools
                       VectorType &                     vector,
                       const ComponentMask &            mask = ComponentMask());
 
+  /**
+   * Like the above function but also taking @p mapping as argument.
```

Comment:
Can you slightly expand this comment, e.g., to say:
```suggestion
   * Like the above function but also taking @p mapping as argument. This might
   * introduce an approximation between the true geometry specified by the
   * manifold if the degree of the mapping is lower than the degree of the finite
   * finite element in the DoFHandler, but more importantly it allows to fill location
   * vectors for mappings that do not preserve vertex locations (like Eulerian
   * mappings).
```
(But give me a few minutes to check the internal workings of the original `get_position_vector` function again.)

Reply:
Thanks, I have added the comment.